tests/test_fields.py

```diff
@@ -2230,14 +2230,33 @@ class ExampleSerializer(serializers.Serializer):
             password = serializers.CharField()
 
             def validate_password(self, obj):
-                err = DjangoValidationError('exc_msg', code='exc_code')
+                err = DjangoValidationError(
+                    'exc_msg %s', code='exc_code', params=('exc_param',),
+                )
```

Comment:
This is not related to the issue, but there was no test that actually used params.